docs/modules/user-guide/partials/proc_developing-with-devfiles.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc_developing-with-devfiles_{context}"]
+= Developing with devfiles
+[role="_abstract"]
+
+Use the devfile specification to develop a node.js “Hello World” application. Developing this application introduces you to how a devfile automates and simplifies your development process.
+
+.Prerequisites
+
+* To develop an application by using the devfile specification, you need to create a Kubernetes cluster on your local machine. To create such a cluster, install link:https://minikube.sigs.k8s.io/docs/start/[minikube].
```

Comment:
Not true. You don't need:
* a *local* cluster to develop an app with devfile spec
* a K8s cluster
* minikube for a local cluster
This should be reformulated to make it clear that this is just a suggested option out of many.

Reply:
Let me know what you think about the revised prereqs!